### DISCUSSION TITLE
Special glass

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4129] [Feature Suggestions]
Add glass that distorts the space when you look through it. (This was in the observation rooms through which the laboratory technicians look after you (If this is feasible))

**Comment by vrad-exe** [comment]
We could add placeable refractive glass, I'm not really sure what purpose it would serve though...

**Reply by ItzAlmaZ** [reply]
I'm not sure myself but it looks better than the usual one

**Reply by SimpleFoxOfficial** [reply]
refractive glass can block laser or would make from 1 laser 2 lasers

**Comment by TeamSpen210** [comment]
It does, but the effect itself is also rather expensive for performance, particularly if you have multiple in front of each other so you can look through more than one.

**Comment by SimpleFoxOfficial** [comment]
I think this will be cool to make for overgrown maps.
